etc/concepts.tsv

```diff
@@ -0,0 +1,336 @@
+Mosquito	Mosquito	1509	MOSQUITO
+Bee	Bee	665	BEE
+Honey	Honey	942	HONEY
+Chamelion	Chamelion	1555	CHAMELEON
+Lizardthecommonone	Lizard (the common one)	632	LIZARD
+Black	Black	163	BLACK
+Rich	Rich	712	RICH
+Igo	I go	695	GO
+Istop	I stop	2880	HALT (STOP)
+Isitdown	I sit down	1649	SIT DOWN
+Ibreathe	I breathe	1407	BREATHE
+Isnore	I snore	1983	SNORE
+Ilaugh	I laugh	1355	LAUGH
+Ifall	I fall	2894	TUMBLE (FALL DOWN)
+Itake	I take	1749	TAKE
+Ilovethee	I love thee	923	LOVE
+Icatchafish	I catch a fish	2638	FISHING
+Ibreakastick	I break a stick	2558	BREAK (CLEAVE)
+Bow	Bow	994	BOW
+Arrow	Arrow	977	ARROW
+Quiver	Quiver	995	QUIVER
+Greegree	Greegree	2804	AMULET
+Sacrifice	Sacrifice	1103	SACRIFICE
+Hell	Hell	878	HELL
+Waterfresh	Water (fresh)		
+Gold	Gold	1369	GOLD
+Book	Book	963	BOOK
+NewMoon	New Moon	3691	NEW MOON
+Soap	Soap	788	SOAP
+Needle	Needle	1382	NEEDLE (FOR SEWING)
+ChainFetters	Chain (Fetters ?)		
+KuskusbearinglikeOats	Kuskus (bearing like Oats)		
+RiceCooked	Rice (Cooked)	1191	COOKED RICE
+Yam	Yam	410	YAM
+Beans	Beans	832	BEAN
+Horse	Horse	615	HORSE
+Butter	Butter	1245	BUTTER
+EweSheep	Ewe (Sheep)		
+RamSheep	Ram (Sheep)	1344	RAM
+Pigeon	Pigeon	1853	DOVE
+Parrot	Parrot	882	PARROT
+Egg	Egg	744	EGG
+Bird	Bird	937	BIRD
+Scorpion	Scorpion	1538	SCORPION
+Butterfly	Butterfly	1791	BUTTERFLY
+Wasp	Wasp	1517	WASP
+Elephant	Elephant	1290	ELEPHANT
+Toad	Toad	894	TOAD
+Frog	Frog	503	FROG
+Littlesmall	Little, small	1246	SMALL
+White	White	1335	WHITE
+Newyoung	New (young)		
+Greedy	Greedy	2018	GREEDY
+Ikneel	I kneel	66	KNEEL
+Irise	I rise	3207	GO UP OR RISE (ONESELF)
+Icutatree	I cut a tree	3151	CUT (WITH AXE)
+Grandfather	Grandfather	1383	GRANDFATHER
+Icallaslave	I call a slave		
+Ipraytogodbeggod	I pray to god (beg god)	24	PRAY
+Grandmother	Grandmother	1496	GRANDMOTHER
+Son	Son	1620	SON
+DaughterMyThy	Daughter (My, Thy)	1357	DAUGHTER
+Toe	Toe	1389	TOE
+Rib	Rib	801	RIB
+Heel	Heel	980	HEEL
+Shoe	Shoe	1381	SHOE
+Trousers	Trousers	809	TROUSERS
+TownVillage	Town (Village)	3609	TOWN OR VILLAGE
+Village	Village	930	VILLAGE
+Market	Market	633	MARKET
+EarRing	Ear-Ring	770	EARRING
+Itch	Itch	148	ITCH
+SmallPox	Small-Pox	1054	SMALLPOX
+GuineaCornbearinglikeMaize	Guinea-Corn (bearing like Maize)		
+Mare	Mare	938	MARE
+Ivory	Ivory		
+Iflogachild	I flog a child		
+Ten	Ten	1515	TEN
+Eleven	Eleven	1706	ELEVEN
+Twelve	Twelve	1707	TWELVE
+Seven	Seven	1704	SEVEN
+Eight	Eight	1705	EIGHT
+Nine	Nine	1483	NINE
+Six	Six	1703	SIX
+Maize	Maize	506	MAIZE
+Twenty	Twenty	1710	TWENTY
+Idol	Idol	1945	IDOL
+Ink	Ink	2643	INK
+Onion	Onion	2366	ONION
+Thirteen	Thirteen	1708	THIRTEEN
+Fourteen	Fourteen	2451	FOURTEEN
+Fifteen	Fifteen	1709	FIFTEEN
+Sixteen	Sixteen	2454	SIXTEEN
+Seventeen	Seventeen	2453	SEVENTEEN
+Eighteen	Eighteen	1713	EIGHTEEN
+Nineteen	Nineteen	1714	NINETEEN
+Buttermelted	Butter (melted)		
+Camwood	Camwood		
+ChainFettersforneck	Chain (Fetters ?) (for neck)		
+Batlarge	Bat (large)		
+Pigeonwild	Pigeon (wild)		
+Yamwild	Yam (wild)		
+Rainwater	Rain (water)		
```

Comment:
Maybe map this one to RAIN (PRECIPITATION) as it clearly points to the substance while the one above map to RAIN OR RAINING?

Reply:
I think this is for food/consumption, e.g. a specific lexeme for water for food from rain.